graphql/user/mutations.ts

```diff
@@ -84,7 +84,17 @@ export class MutateUserResolver {
         section('LOGS', 1);
         result += logs.join('\n');
 
-        logger.info(`An issue was reported from the frontend: \n` + result);
+        const logger = getLogger('IssueReporter');
+        logger.addContext('issureTag', issueTag);
+        logger.addContext('userAgent', userAgent);
```

Comment:
@Jonasdoubleyou any idea what context might be useful from the `GraphQLContext`?

Reply:
The userID, roles and the sessionID, the sessionID is the same we log in all the GraphQL queries.